src/number_theory/zeta_values.lean

```diff
@@ -134,3 +147,214 @@ begin
+begin
+  have : (coe ∘ periodized_bernoulli k : 𝕌 → ℂ) = add_circle.lift_Ico 1 0 (coe ∘ bernoulli_fun k),
+  { ext1 x, refl },
+  rw [this, fourier_coeff_lift_Ico_eq],
+  simpa only [zero_add] using bernoulli_fourier_coeff_eq hk n,
+end
+
+lemma bernoulli_fourier_summable {k : ℕ} (hk : 2 ≤ k) :
+  summable (λ n, -k! / (2 * π * I * n) ^ k : ℤ → ℂ) :=
+begin
+  have : ∀ (n : ℤ), -(k! : ℂ) / (2 * π * I * n) ^ k
+    = (-k! / (2 * π * I) ^ k) * (1 / n ^ k),
+  { intro n, rw [mul_one_div, div_div, ←mul_pow], },
+  simp_rw this,
+  apply summable.mul_left,
+  rw ←summable_norm_iff,
+  have : (λ (x : ℤ), ‖1 / (x:ℂ) ^ k‖) = (λ (x : ℤ), |1 / (x:ℝ) ^ k|),
+  { ext1 x,
+    rw [norm_eq_abs, ←complex.abs_of_real],
+    congr' 1,
+    norm_cast },
+  simp_rw this,
+  rw [summable_abs_iff],
+  exact real.summable_one_div_int_pow.mpr (one_lt_two.trans_le hk),
+end
+
+lemma has_sum_one_div_int_pow_mul_fourier {k : ℕ} (hk : 2 ≤ k) {x : ℝ} (hx : x ∈ Ico (0:ℝ) 1) :
+  has_sum (λ n:ℤ, 1 / (n:ℂ) ^ k * fourier n (x : 𝕌)) (-(2 * π * I) ^ k / k! * bernoulli_fun k x) :=
+begin
+  let B : C(𝕌, ℂ) := continuous_map.mk (coe ∘ periodized_bernoulli k)
+    (continuous_of_real.comp (periodized_bernoulli.continuous (by linarith))),
+  have step1 : ∀ (n:ℤ), fourier_coeff B n = -k! / (2 * π * I * n) ^ k,
+  { rw continuous_map.coe_mk, exact fourier_coeff_bernoulli_eq (by linarith : k ≠ 0) },
+  have step2 := has_pointwise_sum_fourier_series_of_summable ((bernoulli_fourier_summable hk).congr
+    (λ n, (step1 n).symm)) x,
+  simp_rw step1 at step2,
+  convert step2.mul_left ((-(2 * ↑π * I) ^ k) / (k! : ℂ)) using 2,
+  ext1 n,
+  rw [smul_eq_mul, ←mul_assoc, mul_div, mul_neg, div_mul_cancel, neg_neg, mul_pow _ ↑n, ←div_div,
+    div_self],
+  { rw [ne.def, pow_eq_zero_iff', not_and_distrib],
+    exact or.inl two_pi_I_ne_zero, },
+  { exact nat.cast_ne_zero.mpr (nat.factorial_ne_zero _), },
+  { rw [continuous_map.coe_mk, function.comp_app, of_real_inj,
+      periodized_bernoulli, add_circle.lift_Ico_coe_apply (by rwa zero_add)] },
+end
+
+end bernoulli_periodized
+
+section cleanup
+/- This section is just reformulating the results in a nicer form. -/
+
+lemma has_sum_one_div_nat_pow_mul_fourier {k : ℕ} (hk : 2 ≤ k) {x : ℝ} (hx : x ∈ Ico (0:ℝ) 1) :
+  has_sum (λ n:ℕ, 1 / (n:ℂ) ^ k * (fourier n (x : 𝕌) + (-1) ^ k * fourier (-n) (x : 𝕌)))
+  (-(2 * π * I) ^ k / k! * bernoulli_fun k x) :=
+begin
+  convert (has_sum_one_div_int_pow_mul_fourier hk hx).sum_nat_of_sum_int',
+  { ext1 n,
+    rw [int.cast_neg, mul_add, ←mul_assoc],
+    conv_rhs { rw [neg_eq_neg_one_mul, mul_pow, ←div_div] },
+    congr' 2,
+    rw [div_mul_eq_mul_div₀, one_mul],
+    congr' 1,
+    rw [eq_div_iff, ←mul_pow, ←neg_eq_neg_one_mul, neg_neg, one_pow],
+    apply pow_ne_zero, rw neg_ne_zero, exact one_ne_zero, },
+  { rw [int.cast_zero, zero_pow (by linarith : 0 < k), div_zero, zero_mul, add_zero] },
+end
+
+lemma has_sum_one_div_nat_pow_mul_cos {k : ℕ} (hk : 1 ≤ k) {x : ℝ} (hx : x ∈ Ico (0:ℝ) 1) :
```

Comment:
`hk` should be `k ≠ 0`. Also, isn't this also true for `x = 1`, i.e., should `hx` be `x ∈ Icc (0:ℝ) 1`?